path/__init__.py

```diff
@@ -317,13 +267,26 @@ def getcwd(cls):
         """
         return cls(getcwdu())
 
+    cwd = classmethod(getcwd)
+
+    @classmethod
+    def home(cls):
+        """ This class method works like pathlib "Path.home" class method.
```

Comment:
Docstring should describe what it does, s

```suggestion
        """ Return a Path pointing to the user's home directory.
```